app/common/utils.js

```diff
@@ -27,4 +29,11 @@ module.exports = {
     isDevEnv() {
         return process.env.NODE_ENV === "dev";
     },
+    collapseHeader($el) {
```

Comment:
I like the idea of creating this function, but wouldn't it make more sense having this in materialize (or maybe elsewhere)?

keep in mind that utils.js (and everything under common) is intended to be used (or at least, to not depend) on either renderer or main proceses, while `collapseHeader` is exclusively a renderer process